src/components/addBooks.js

```diff
@@ -63,7 +67,7 @@ const AddBook = ({ onSubmit }) => {
           <option value="Drama">Drama</option>
           <option value="Si-Fi">Si-Fi</option>
         </select>
-        <button type="submit">Add</button>
+        <button className="add-btn" type="submit">Add</button>
       </form>
```

Comment:
- [ ] Please change the color of your button text because it's not the same as [Zaplin](https://zeplin.io/) design and also change the text with `ADD BOOK` of your button :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-25 03-22-11](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/59ec3c0f-5f46-4578-bbc8-368245c72125)

**Zaplin Design** 
![Screenshot from 2023-06-25 03-22-03](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/70a54a8a-b6dc-4332-b7bf-efcd8e139156)